README.markdown

```diff
@@ -48,7 +48,9 @@ Note: On OSX, if you get `nvm: command not found` after running the install scri
 
 To verify that nvm has been installed, do
 
-    command -v nvm
+`source ~/.bash_profile` (or whichever profile you added nvm to)
```

Comment:
Hrmmm thanks for the feedback. It seems to be installed on my machine though...but I `| zsh` instead of `| bash`